client/blocks/edit-gravatar/index.jsx

```diff
@@ -215,6 +214,7 @@ export class EditGravatar extends Component {
 						</div>
 					</FilePicker>
 				</div>
+				{ this.renderImageEditor() }
```

Comment:
FYI, I have #36656 out with this change already.

Reply:
And it's now merged, thanks to your review, so a rebase will pull it in 👍

Reply:
Rebased, and the commit magically disappeared :)